libcloud/compute/drivers/gce.py

```diff
@@ -2080,7 +2080,7 @@ def ex_set_image_labels(self, image, labels):
             raise ValueError("Must specify a valid libcloud image object.")
         current_fp = image.extra['labelFingerprint']
         body = {'labels': labels, 'labelFingerprint': current_fp}
-        request = '/global/%s/setLabels' % (image.name)
+        request = '/global/images/%s/setLabels' % (image.name)
```

Comment:
Just adding this for reference here - https://cloud.google.com/compute/docs/reference/rest/v1/images/setLabels